exonum-java-binding/testkit/src/test/java/com/exonum/binding/testkit/TestKitTest.java

```diff
@@ -211,38 +216,41 @@ void setInvalidValidatorCount() {
     Class<IllegalArgumentException> exceptionType = IllegalArgumentException.class;
     short invalidValidatorCount = 0;
     TestKit.Builder testKitBuilder = TestKit.builder()
-        .withService(TestServiceModule.class);
+        .withDeployedService(ARTIFACT_ID, ARTIFACT_FILENAME)
+        .withService(ARTIFACT_ID, SERVICE_NAME, SERVICE_ID,
+            Any.getDefaultInstance());
     assertThrows(exceptionType, () -> testKitBuilder.withValidators(invalidValidatorCount));
   }
 
   @Test
   void requestWrongServiceClass() {
     Class<IllegalArgumentException> exceptionType = IllegalArgumentException.class;
     try (TestKit testKit = TestKit.builder()
-        .withService(TestServiceModule.class)
+        .withDeployedService(ARTIFACT_ID, ARTIFACT_FILENAME)
+        .withService(ARTIFACT_ID, SERVICE_NAME, SERVICE_ID,
+            Any.getDefaultInstance())
         .build()) {
       assertThrows(exceptionType,
-          () -> testKit.getService(TestService.SERVICE_ID, TestService2.class));
+          () -> testKit.getService(SERVICE_NAME, TestService2.class));
     }
   }
 
   @Test
-  void requestWrongServiceId() {
+  void requestWrongServiceId(TestKit testKit) {
     Class<IllegalArgumentException> exceptionType = IllegalArgumentException.class;
-    try (TestKit testKit = TestKit.forService(TestServiceModule.class)) {
-      assertThrows(exceptionType, () -> testKit.getService((short) -1, TestService2.class));
-    }
+    assertThrows(exceptionType, () -> testKit.getService("Invalid service name",
+        TestService.class));
```

Comment:
Why the class changed?

Reply:
I think that there was a typo initially, as TestKit was created with `TestService` and was checking that it would throw if a service with an invalid serviceId was requested. The service class doesn't matter that much, given the serviceId (serviceName in new implementation) is incorrect, but still it should be `TestService`.